src/components/printState/PrintState.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { StyleSheet, Text, View, ViewStyle } from 'react-native';
+import { useTheme } from '../..//styles/themes';
+import { Button } from '../buttons/Button';
+import { Icon } from '../icons/Icon';
+
+interface Props {
+    text: string;
+    buttonLabel: string;
+    onPress: () => void;
+    style?: ViewStyle;
+}
+
+export const PrintState = ({ text, buttonLabel, onPress, style }: Props) => {
+    const theme = useTheme();
+
+    const styles = StyleSheet.create({
+        container: {
+            width: '100%',
```

Comment:
I think it ends up to the same result, but just for consistency, could we keep using flexbox layout `flex:1` instead of percentage `100%` ?

Cf: https://reactnative.dev/docs/height-and-width#flex-dimensions

Reply:
Fine, we keep width:100% i need to dig into it but I don't have the time now...